lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -321,6 +321,9 @@ namespace Js
                 return FALSE;
             JavascriptError::ThrowTypeError(scriptContext, JSERR_ErrorOnRevokedProxy, _u("get"));
         }
+
+        RecyclableObject *target = this->target;
```

Comment:
According to ECMA262 9.5.5, [[GetOwnProperty]](P) must assign proxy target to a local variable 'target' and do business with the same.


---
In reply to: [68428440](https://github.com/Microsoft/ChakraCore/pull/1195#discussion_r68428440) [](ancestors = 68428440)